packages/repository-json-schema/docs.json

```diff
@@ -0,0 +1,8 @@
+{
+  "content": ["index.ts", "src/index.ts", "src/build-schema.ts"],
```

Comment:
We usually put each content file on its own line, to make future patches smaller.

```json
{
  "content": [
    "index.ts",
    "etc."
  ],
  "etc."
}
```